src/main/java/com/labsynch/labseer/chemclasses/bbchem/CmpdRegMoleculeBBChemImpl.java

```diff
@@ -76,8 +76,9 @@ public String getMolStructure() throws CmpdRegMolFormatException {
 
 	@Override
 	public String getFormula() {
-		if(this.getRegistrationStatus() == RegistrationStatus.ERROR) return null;
-		if(this.molecule.getRegistrationStatus() != RegistrationStatus.ERROR && this.molecule.getMolecularFormula() != null) {
+		if(this.getRegistrationStatus() == RegistrationStatus.ERROR) { 
+			return null; 
+		} else if(this.molecule.getRegistrationStatus() != RegistrationStatus.ERROR && this.molecule.getMolecularFormula() != null) {
```

Comment:
`this.molecule.getRegistrationStatus() != RegistrationStatus.ERROR &&` is redundant. This can just be
`else if(this.molecule.getMolecularFormula() != null`

Reply:
Thanks, missed this just pushed. I'll remerge this branch as I jumped the gun

Reply:
Github doesn't let you re-open a pull request so I just just created a new one and merged it: https://github.com/mcneilco/acas-roo-server/pull/259